tests/check_codeblocks.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Script that checks if the Code::Blocks project is complete.
+# TODO: If the project is incomplete generate a patch file to make it complete.
+
+# Determine path of the current script, and go to the ES project root.
+HERE=$(cd `dirname $0` && pwd)
+cd ${HERE}/..
+
+ESTOP=$(pwd)
+CDPROJECT=${ESTOP}/EndlessSkyLib.cbp
+CDTPROJECT=${ESTOP}/EndlessSkyTests.cbp
+
+RESULT=0
+
+for FILE in $(find source -type f -not -name WinApp.rc -not -name main.cpp | sed s,^source/,, | sort)
+do
+  # Check if the file is already in the general Code::Blocks project.
+  if ! fgrep -q "${FILE}" ${CDPROJECT}; then
+    if [ $RESULT -ne 1 ]; then
+      echo -e "\033[1mMissing files in EndlessSkyLib.cbp:\033[0m"
+    fi
+    echo -e "${FILE}"
+    RESULT=1
+  fi
+done
+
+for FILE in $(find tests/src/ -type f -name "*.h" -o -name "*.cpp" | sed s,^tests/src/,, | sort)
```

Comment:
We probably need to include the files in the other tests subdirectories as well

Reply:
What other tests subdirectories are you referring to? If you mean the ones in `tests/src` then those are already included since `find` is recursive.

Reply:
I was thinking of the include directory but we only put headers in it